tools/WebStack.versions.settings.targets

```diff
@@ -4,8 +4,8 @@
   <!-- Set the version number: major, minor, build and release (i.e. alpha, beta or blank for RTM)-->
   <PropertyGroup>
     <VersionMajor Condition="'$(VersionMajor)' == ''">7</VersionMajor>
-    <VersionMinor Condition="'$(VersionMinor)' == ''">5</VersionMinor>
-    <VersionBuild Condition="'$(VersionBuild)' == ''">14</VersionBuild>
+    <VersionMinor Condition="'$(VersionMinor)' == ''">7</VersionMinor>
```

Comment:
I will revert the version change to another release PR to change the version.

Reply:
I don't understand this change. Also, why wouldn't it be `7.6.0` or `7.5.15`?

Reply:
You could have left as it is and only change in the release PR

Reply:
As i mentioned, i will revert this change in this PR and will have another pr to update the version.